components/member-profile/contribution/index.js

```diff
@@ -107,7 +107,9 @@ const ContributionCard = ({ contribution, fullName, imageLink, devUser, url, url
             {completedText}
             <p className={classNames.completedDate}>{completedDate}</p>
           </div>
-          <div className={classNames.featureLiveOnText}>{featureLiveOnText}</div>
+          <div className={classNames.featureLiveOnText}>
+            {featureLiveOnText && <span>Feature live on {featureLiveOnText}</span>}
+          </div>
```

Comment:
Let's wrap the `div` which is on line no 110 under the condition `featureLiveOnText`

because what if `featureLiveOnText` is not true? In that case, we will be rendering a `div` that will have no content.

```javascript
{featureLiveOnText && (<div><span>...content</span></div>){
```

Reply:
Checked and fixed this.